src/config.c

```diff
@@ -840,6 +857,10 @@ RSConfigOptions RSGlobalConfigOptions = {
                      "high memory consumption.",
          .setValue = setCursorMaxIdle,
          .getValue = getCursorMaxIdle},
+        {.name = "INDEX_CURSOR_LIMIT",
+         .helpText = "max number of cursors that can be opened in the context of a single index",
```

Comment:
per shard, in the case of cluster mode, correct? consider mentioning that.

Reply:
Yep, it is just that up until now this was implicit in all of our configuration parameters.
e.g `CURSOR_MAX_IDLE` or any other shard configuration.
Don't think we should deviate from the convention.